.cicd/jobs/lint_and_test.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
-#Install uwtools
-pip install -e .
+# Cleanup old messes (should be removed after one run of the pipeline)
```

Comment:
Is there any way we can set a task in Jenkins to run this instead of adding this then removing it before it runs again?

Reply:
OK good. I think I remember that you can run a script outside of the pipeline, but I'm not sure, still waiting on access, myself.